cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/Content/DetailsInfo/DetailsInfo.scss

```diff
@@ -57,14 +58,22 @@ $details-row-border-color: $grey-05;
           &:nth-child(3) {
             border-top: 1px solid $details-row-border-color;
           }
+
+          > .label-holder {
+            width: $label-width;
+            max-width: $label-width;
```

Comment:
is this trying to overwrite something? max-width shouldn't be needed, or set this to display inline-block